umpf

```diff
@@ -928,7 +928,10 @@ parse() {
 		test -n "${content}" || bailout "${cmd} line without value"
 		if ${STABLE}; then
 			local name="$(<"${STATE}/name")"
-			local version="$(sed "s;${name}/\([0-9]\{8\}\)-.*;\1${VERSION_SEPARATOR}${VERSION};" <<< "${content}")"
+			if [ "${namespace}" = "format_patch" ]; then
+				VERSION="$(sed "s;${name}/[0-9]\{8\}${VERSION_SEPARATOR}\(.*\);\1;" <<< "${content}")"
```

Comment:
Note that this also makes it impossible to get something like `20240314-4` or similar from a 20240314-1 tag by calling `umpf format-patch -v4`.